examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -186,10 +186,14 @@ def compute_prediction(X, model_name):
             name=model_name,
             linewidth=linewidth,
             ax=axs[i // cols, i % cols],
+            plot_chance_level=(model_idx == len(models_name) - 1),
+            chance_level_kwargs={
+                "linewidth": linewidth,
+                "linestyle": ":",
+            },
         )
-    axs[i // cols, i % cols].plot([0, 1], [0, 1], linewidth=linewidth, linestyle=":")
     axs[i // cols, i % cols].set_title(dataset_name)
-    axs[i // cols, i % cols].set_xlabel("False Positive Rate")
-    axs[i // cols, i % cols].set_ylabel("True Positive Rate")
 plt.tight_layout(pad=2.0)  # spacing between subplots
 plt.show()
+
+# %%
```

Comment:
Please remove this marker as previously suggested.

Reply:
Sorry, I have made but not pushed these previous changes. I will push all of those together soon.

Reply:
No problem. I was too quick to look at the PR :).